README.md

```diff
@@ -1,6 +1,8 @@
 # jag-bcsc-auth-service
 BCSC Auth service (eCRC microservice) by Ministry of Justice
 
+BCSC Auth service API utilizes OIDC library to integrate with the BC Services Card (BCSC) to verify user identity. 
```

Comment:
```suggestion
BCSC Auth service API utilizes the OIDC library to integrate with the BC Services Card (BCSC) to verify user identity. 
```

Reply:
this is a good concise summary; however, reading into the issue ticket maybe we need to provide more details about what the service is? not too sure what else to mention though :)

Reply:
maybe I should provide more info about the different endpoints? let me see what else I can add.

Reply:
sounds like a good idea! and also some additional background on information about dependencies on other services being setup and configured (ie. BC Services Card configuration)

Reply:
I have added everything that I can think of